tutor/templates/build/openedx/Dockerfile

```diff
@@ -181,8 +181,14 @@ COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
 
 # Pull latest translations via atlas
-RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --branch='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} \
-    translations/edx-platform/conf/locale:conf/locale
+RUN mkdir -p conf/plugins-locale/plugins conf/plugins-locale/xblock.v1/ lms/static/js/xblock.v1-i18n cms/static/js
+RUN ./manage.py lms --settings=tutor.i18n pull_plugin_translations --verbose --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }}
```

Comment:
Good point. Tutor doesn't have to be aware of those details.

> Is that possible? Let me know if you want to make that upstream change now or in a later PR. (or never)

It's possible to avoid having Tutor aware of the details if we use `make ATLAS_OPTIONS="{{ ATLAS_OPTIONS }}" pull_translations` in the Tutor edx-platform Dockefile. 

That seems like a good way to fix it and later we can refactor the platform Makefile and Django commands if needed.

There's a lot of small details that we haven't covered in FC-0012 to avoid scope creep. We haven't yet seen `atlas` used in production and I'd like to focus energy on that as opposed to perfect what we have before operators (aka customer) feedback.

Nevertheless I'm open to the `make ATLAS_OPTIONS="{{ ATLAS_OPTIONS }}" pull_translations` idea if you think adding another level of abstraction is worth it.

Reply:
> There's a lot of small details that we haven't covered in FC-0012 to avoid scope creep. We haven't yet seen atlas used in production and I'd like to focus energy on that as opposed to perfect what we have before operators (aka customer) feedback.

I'm totally fine with that, as long as we finalise everything before the Redwood cut.

For the record, my comment did not as much concern the use of the `manage.py` commands, but more the detailed list of destination directories. In other words: I'm fine with the `manage.py` commands, but I would like to get rid of the `mkdir -p ...` command.

Reply:
You're right. Those `mkdir -p` belong to Python. I think we can move these commands into Python.

Here's the options I have ordered by time:

 1- Merge as-is and issue new PRs to clean up both edx-platform and Tutor
 2- Use `make ATLAS_OPTIONS="{{ ATLAS_OPTIONS }}" pull_translations` and clean up the `edx-platform` asynchronously
 3- Wait until a new edx-platform clean up PR is merged then update this PR and merge

My preferred option is (2) because it would keep the concerns in `edx-platform` and maybe someone (me?) will clean it.

Reply:
@regisb You're right. Those `mkdir -p` belong to Python. I think we can move these commands into Python.

Here's the options I have ordered by time:

 1- Merge as-is and issue new PRs to clean up both edx-platform and Tutor
 2- Use `make ATLAS_OPTIONS="{{ ATLAS_OPTIONS }}" pull_translations` and clean up the `edx-platform` asynchronously
 3- Wait until a new edx-platform clean up PR is merged then update this PR and merge

My preferred option is (2) because it would keep the concerns in `edx-platform` and maybe someone (me?) will clean it.

Reply:
@regisb as you know I'm really eager to this merged.

I've created an `edx-platform` PR which address your concerns:

 - https://github.com/openedx/edx-platform/pull/34257

I'd love for this to be merged and then I'll make a tiny clean up PR on Tutor.

We've issued so far 250+ PRs on the FC-0012 project and there's quite of a lag everywhere and I appreciate your support on this one.

Reply:
I very much appreciate you opening the upstream PR. We can either:

1. Wait for [this PR](https://github.com/openedx/edx-platform/pull/34257) to be merged before we merge this one.
2. Add a `REMOVE-AFTER-V18` comment above line 184 and then merge this PR.

I'm fine with both. Let me know what you prefer.

Reply:
More things is coming up such as:

 - https://github.com/openedx/edx-platform/pull/34261

Reply:
Please let me know how you want to proceed.

Reply:
> Wait for https://github.com/openedx/edx-platform/pull/34257 to be merged before we merge this one.

@regisb the other pull request is already merged. This pull request has been unblocked.


> https://github.com/openedx/edx-platform/pull/34261

This one has been duplicated by adding [one more line](https://github.com/overhangio/tutor/pull/993/files#diff-ebe2aa009411bb8f1a30abe8461ed746d5c8f5869aec9ec7e9221b4ff40960d1R188). So it's not blocked by anything anymore.

I'm happy to say this pull request is ready :)